domains/certificates/Query.API/API/Controllers/CertificatesController.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Linq;
+using API.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[ApiController]
+public class CertificatesController : ControllerBase
+{
+    [HttpGet]
+    [Route("v1/certificates")]
```

Comment:
Are we sure about this route?

Reply:
We have not discussed/designed the API upfront. At this point we only need a single endpoint/route, so I thought this was the simplest and also gave some versioning.

Reply:
After talking with Danni we found that having "v1" in the route will be strange wrt. the ingressroute. Since API versioning is not in place in general on Energioprindelse, we will remove it.